themes/classic/modules/ps_shoppingcart/modal.tpl

```diff
@@ -59,15 +59,15 @@
               <p><span class="label">{l s='Subtotal:' d='Shop.Theme.Checkout'}</span>&nbsp;<span class="value">{$cart.subtotals.products.value}</span></p>
               <p><span>{l s='Shipping:' d='Shop.Theme.Checkout'}</span>&nbsp;<span class="value">{$cart.subtotals.shipping.value} {hook h='displayCheckoutSubtotalDetails' subtotal=$cart.subtotals.shipping}</span></p>
 
-              {if !$configuration.display_prices_tax_incl}
+              {if $cart.totals.total_including_tax.value != $cart.totals.total_excluding_tax.value && $cart.totals.total.value == $cart.totals.total_excluding_tax.value}
```

Comment:
Why did you revert this?

Reply:
It was creating an issue when you disable tax on the shop. Total tax excluded and total tax included were on every displayed. Only Total tax excluded was supposed to be displayed. I put it back to fix but I'm working with your code to fix the issue